Backend/Remora.Discord.API/API/CDN.cs

```diff
@@ -409,6 +409,75 @@ public static Result<Uri> GetGuildBannerUrl
         return ub.Uri;
     }
 
+    /// <summary>
+    /// Gets the CDN URI of the given user's avatar decoration.
+    /// </summary>
+    /// <param name="user">The user.</param>
+    /// <param name="imageFormat">The requested image format.</param>
+    /// <param name="imageSize">The requested image size. May be any power of two between 16 and 4096.</param>
+    /// <returns>A result which may or may not have succeeded.</returns>
+    public static Result<Uri> GetUserAvatarDecorationUrl
+    (
+        IUser user,
```

Comment:
`IPartialUser` might be better to use to also support those instances.

Reply:
It's `IUser` to be consistent with the other methods (such as GetUserBannerUrl below)

Reply:
In general I think `IPartialUser` should be used.

Reply:
I think that'd be better suited for its own PR to overhaul the entire CDN class. cc @Nihlus?

Reply:
Yeah, let's do that separately.